index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" type="image/*" href="assets/images/portfolio.png">
+    <link rel="stylesheet" href="./assets/style/main.css">
+    <title>First Mobile version Portfolio</title>
+</head>
+<body>
+        <header class="header-content">
+            <span><a href="#"  id="mylogo" class="common-font-family">My Logo</a></span> 
+            <nav id="nav-ham-icon"><span id="hamburgur-icon">&#9776;</span></nav>
+        </header>
+        <main class="wrap-content">
+                <h1 id="myfullname" class="common-font-family">I’m Osman Wako<br>
+                    Glad to see you!</h1>
+                <p class="common-font-family">
+                    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience!
+                     If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                    <button type="button" id="btn-letconid" class="btn-letcon common-font-family">LET’S CONNECT</button>
+                <ul class="social-md">
+                    <li><a href="#"><img src="assets/images/social-icon/twitter-icon.png"></a></li>
+                    <li><a href="https://www.linkedin.com/in/osman-wako-08663825a/"><img src="assets/images/social-icon/linkedin icon.png"></a></li>
+                    <li><a href="https://mail.google.com/"><img src="assets/images/social-icon/m-icon.png"></a></li>
+                    <li><a href="https://github.com/osmanwako"><img src="assets/images/social-icon/github-icon.png"></a></li>
+                    <li><a href="#a"><img src="assets/images/social-icon/clip-icon.png"></a></li>
+                </ul>
+        </main>
+</body>
+</html>
```

Comment:
- [ ] Also, this request was made by the previous reviewer to alighn your content just as it is on the figma design. I've attached a picture comparison for you. this helps give a better user experience to your project viewers 👍 

## Your view:
![image](https://user-images.githubusercontent.com/50990925/219554049-3e935148-8d19-4638-b482-ecc286f1d817.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/219554104-793b8210-52e8-4352-9cb7-a85e78eed498.png)